Forms/Main.cs

```diff
@@ -826,7 +827,7 @@ private void ShowDownloadInfo(int epid)
             }
 
             this.SetToolbarButtons(buttons.ToArray());
-            this.SetSideBar(TextUtils.StripDateFromName(info.Name, info.Date), infoText, Model.Episode.GetImage(epid));
+            this.SetSideBar(TextUtils.StripDateFromName(progInfo.Name + " - " + info.Name, info.Date), infoText, Model.Episode.GetImage(epid));
```

Comment:
The programme name shouldn't be passed to `StripDateFromName`as programme names don't tend to contain dates.  I'd also personally favour a colon and a space as the separator.